redbot/cogs/mod/kickban.py

```diff
@@ -218,7 +232,22 @@ class KickBanMixin(MixinMeta):
 
         If days is not a number, it's treated as the first word of the reason.
         Minimum 0 days, maximum 7. Defaults to 0."""
-
+        author = ctx.author
+        guild = ctx.guild
+        if reason == None:
+            reason = "No reason was given."
+        else:
+            reason = reason
+        self == self.bot.get_cog("Mod")
+        toggle = await self.settings.guild(guild).toggle_dm()
+        if toggle == True:
+            with contextlib.suppress(discord.HTTPException):
+                em = discord.Embed(
+                    title=_("**You have been banned from {guild}.**").format(guild=guild)
+                )
+                em.add_field(name=_("**Moderator**"), value=author.name, inline=False)
```

Comment:
Showing the reason here is fine. Showing the moderator is not unless there is another setting for this.

Reply:
Would you prefer no moderator or an extra setting for it?